SparkPost/matchbox
# UX-458 Remove spreading of unknown Toggle props

### What Changed
- Removes spreading of unknown Toggle props

### How To Test or Verify
- Verify the app does not use props that are not defined:
  - https://proply-2web2ui.vercel.app/
  - Verify `Toggle` and `ToggleBlock`

### PR Checklist
